parity/secretstore/server.rs

```diff
@@ -134,6 +134,11 @@ mod server {
 
 	const SECP_TEST_MESSAGE: H256 = H256([1_u8; 32]);
 
+	/// Version of the secret store database
+	const SECRET_STORE_DB_VERSION: &str = "4";
```

Comment:
I think it should be a u8 as in https://github.com/openethereum/openethereum/pull/11322/files#diff-8a98b639ef87df303b25d0aca513a7a0R31 (not that it matters much, but it's nice to be consistent)

Reply:
I can change it but i would have to convert it to a `&str`
`format!("{}", SECRET_STORE_DB_VERSION).as_bytes()`

Reply:
could it be `&[SECRET_STORE_DB_VERSION]`?

Reply:
no that doesn't write the string "4" to the file.

Reply:
They do `format!()` in https://github.com/openethereum/openethereum/pull/11322/files#diff-8a98b639ef87df303b25d0aca513a7a0R162

Reply:
right, it was me who wrote that code :) Now that I think about it, it's something that could be done in `open_secretstore_db` function which lives https://github.com/paritytech/secret-store,

but for now, this fix is ok

Reply:
Are there still new version of SecretStore 1.0?

Reply:
If the question is whether we're planning to release a version 1.0.1, we could create a branch for 1.x for secret store, yes.